Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Status page OpenAPI decision
+type: Decision Record
+status: Draft
+track: Ops
+---
+
+## Introduction
+
+While defining the [OpenAPI spec](https://github.com/SovereignCloudStack/status-page-openapi) some considerations and decisions are made and should be documented.
+
+## Requirements
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+In addition, "FORBIDDEN" is to be interpreted equivalent to "MUST NOT".
+
+## Motivation
+
+The spec should be as minimal as possible, while being as understandable as possible, so some choices to the design of API objects, requests and responses are made.
+
+## Decision
+
+### Common values
```

Comment:
```suggestion
### Common definitions
```

In my thinking, a value would be `"astringvalue"` while the definition may be `Id (type string)`. "Value" in this context definitely confused me - it sounds like enum's or similar are used.